### ISSUE TITLE
Feature request: copy-to-clipboard button for code blocks

### ISSUE THREAD

**Issue by ZiriaRMD** [zadam/trilium#1980]
It would be nice to have a button to copy an entire code block to the clipboard:

![image](https://user-images.githubusercontent.com/52488338/119777405-64dded80-bec6-11eb-919c-b01a7e37597d.png)

**Comment by zadam** [comment]
Hi, yes, this would be nice. There has been similar requests made in the past - e.g. #973

This is a CKEditor feature, so I created a feature request there: https://github.com/ckeditor/ckeditor5/issues/9774

**Comment by root9191** [comment]
Hello, 
are there any Updates yet for this Feature?

**Comment by nekrondev** [comment]
@root9191 looks like a dependency on ckeditor which hasn't yet included copy to clipboard for code blocks.

**Comment by ZiriaRMD** [comment]
Yes you can go to https://github.com/ckeditor/ckeditor5/issues/9774 and give a 👍to the request

**Comment by thanayut1750** [comment]
We want this feature!!!! any update ?

**Comment by zhdenny** [comment]
Damn, came here looking to see if others wanted this. I hope we see this feature included in Trilium soon. Sweet application.

**Comment by Nriver** [comment]
Refer to https://github.com/Nriver/copy-code-block-widget
